packages/fxa-settings/src/pages/SigninConfirmed/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import SigninConfirmed from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'pages/SigninConfirmed',
+  component: SigninConfirmed,
+} as Meta;
+
+export const Default = () => (
```

Comment:
Not sure if it's worth adding this state now but if I just hit the page locally, I see this text:

<img width="682" alt="image" src="https://user-images.githubusercontent.com/13018240/209202838-622d9bd8-c6d1-4b97-8531-890bffe0de3d.png">

I think the image is bigger here than it is in prod too, we can pass a `w-#` TW class to the SVG to modify.

Reply:
Oooh good catch with the image size -- I'm against adding this state in since I don't want to add in all of the Ready component edge cases unless we're sure we need them, but happy to do so if you feel strongly!

Reply:
Hmmmmmm weirdly the images are looking the same size to me in prod + local, but seem larger in Storybook...

Reply:
The storybook acceptance criteria just says to add at least one state, so it's not necessary for r+. It looks like we'll definitely need this one though, since I can see it by starting up the stack fresh and hitting that page.

> Hmmmmmm weirdly the images are looking the same size to me in prod + local, but seem larger in Storybook...

Oh yeah sorry, that's what I meant. The Backbone page has the image smaller, probably because it's loaded as a background-image, so we should update the React version to match a similar width size. There might be some trial and error on the right class name/width to make it visually look the same because the viewbox of the SVG is creating a lot of whitespace on either side so it's not as easy as "looks like 128px, find the TW width name that matches", it'll have to be higher than that. You can pass the class directly on the SVG.